### ISSUE TITLE
NASA Recognition on 889 Tool

### ISSUE THREAD

**Issue by rebekahperillo** [GSA/smartpay-website#201]
NASA, as the original creator of the 889 tool, needs to be given more credit on the 889 tool.

https://889.smartpay.gsa.gov/#/

Acceptance Criteria:
- Place the NASA logo (file below) on the top of the page with the GSA logo. 
- Replace current sentence "This tool only checks for Section 889 representations in an entity's SAM.gov record." with "This collaborative tool, developed by NASA and GSA, only checks for Section 889 representations in an entity's SAM.gov record."

![Image](https://github.com/GSA/smartpay-website/assets/133424018/a04d1879-1fe5-48c7-aed1-7e391a0f5864)

@JessicaMarine1 How does this look?

**Comment by rebekahperillo** [comment]
These all look great. Do you mind if I pass them along to Jeff White from
OSC to hear his thoughts? Then, I'll pass along to my supervisor for final
weigh in.

Does this plan for the team?

Rebekah Knouse Perillo (she/her)

Lead Business Management Specialist

Center for Charge Card Management
Office of Professional Services and Human Capital Categories
Federal Acquisition Service, U.S. General Services Administration

301-938-5257

***@***.***


Follow the GSA SmartPay program on social media...Tweet with us on Twitter
<https://twitter.com/GSASmartPayNews> and friend us on Facebook
<https://www.facebook.com/GSASmartPayNews/>!
[image: image.png]

**Comment by JessicaMarine1** [comment]
Feedback from Jeff White (OSC branding):

Version 1 - not feeling the reverse Star Mark next to the full color NASA logo (and the distance between the two needs to be increased so that there is one full Star Mark  between them). The blue of the NASA logo is also getting lost in the background. Since NASA doesn't have a reverse version of their logo we should use our full color version of the Star Mark and make the header of the page white (following our normal guidelines for white bar size found in our branding guidelines).

Version 2 - need to increase the space between the Star Mark and text to allow for one full Star Mark between them

Version 3 - same feedback for text as in v.2 and the mix of one color and full color doesn't work. Change headers background color to white and go full color for both logos.

**Comment by klohman** [comment]
## Updated 
### Option 1
![image](https://github.com/GSA/smartpay-website/assets/18366192/39ba23e7-8cc6-42cb-abf8-008c30bc7f74)

### Option 2 
![image](https://github.com/GSA/smartpay-website/assets/18366192/300e728f-bace-46b3-b6bd-57733790b853)


### Option 3
![image](https://github.com/GSA/smartpay-website/assets/18366192/faae5caa-ab1b-4d0e-9043-ba6d266ae717)

**Comment by JessicaMarine1** [comment]
Feedback from Sarah Bryant in OSC:
1. Use the full USWDS identifier component, including the required links. And it must say "An official website of the U.S. General Services Administration".
2. Consider the site name and description. The "889 compliance search" title implies a user can enter criteria and get a set of results. But it appears the functionality is actually to check if a specific vendor is 889 compliant.  
3. The "889 Compliance SAM Tool web application" is bureaucratese. A plain language version would be along the lines of "This tool was built by NASA using the [sam.gov](http://sam.gov/) Entity Management API".
4. If the site isn't being co-managed with NASA, but has been handed over to GSA to manage, it makes more sense for the partnership information to be in the footer, as presented in option 2.


TLDR:
- Recommends Option 2 (attribution in footer)
- Need to incorporate standard GSA footer
- Reconsider wording of 1st sentence in gray footer
- Reconsider name of tool

**Comment by klohman** [comment]
Regarding the comment on the name of the tool, I noticed the live site has it called "889 Representations Search" (the old wireframes I pulled had the other name). 

Here's an updated mockup that more closely matches the live site. This mock also includes: 
- the NASA logo
- the updated footer 
- updated language : "This tool was built by NASA using the sam.gov Entity Management API"

![image](https://github.com/GSA/smartpay-website/assets/18366192/e7ef0e46-5015-45db-bde2-9e04d361afff)

**Comment by JessicaMarine1** [comment]
@klohman could you update the light gray footer to:

This tool was built by NASA using the SAM.gov Entity Management API.

We encourage input! Please provide feedback and suggestions to [gsa_smartpay@gsa.gov](mailto:gsa_smartpay@gsa.gov)

[Limitation on Permissible Use of Dun & Bradstreet, Inc. (D&B) Data](https://sam.gov/content/about/disclaimers)

**Comment by klohman** [comment]
@JessicaMarine1 Updated. 

![image](https://github.com/GSA/smartpay-website/assets/18366192/8b2f8d12-4d28-4922-ad30-710e9c9f5e8e)

**Comment by JessicaMarine1** [comment]
Update 8/2/2023: 

- Rebekah provided [mockup](https://user-images.githubusercontent.com/18366192/257282577-8b2f8d12-4d28-4922-ad30-710e9c9f5e8e.png) to Erin to review with her contact at NASA. 
- Reminded Erin about NASA's [Media Usage Guidelines](https://www.nasa.gov/multimedia/guidelines/index.html)
- Recommended holding the implementation of any changes until the new sites go live.

**Comment by JessicaMarine1** [comment]
We need to put the GSA footer on the 889 tool regardless of the NASA co-branding decisions, so I'm going to make that another story.

**Comment by rebekahperillo** [comment]
@JessicaMarine1 @mark-meyer Design approved by NASA! Also are we able to update the light gray portion to have a period after gsa_smartpay@gsa.gov? Thank you!

**Comment by JessicaMarine1** [comment]
Please save this work for KL&A :)
